apps/emqx_bridge_pulsar/src/emqx_bridge_pulsar_sup.erl

```diff
@@ -0,0 +1,24 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2023 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+-module(emqx_bridge_pulsar_sup).
+
+-behaviour(supervisor).
+
+-export([start_link/0]).
+
+-export([init/1]).
+
+-define(SERVER, ?MODULE).
+
+start_link() ->
+    supervisor:start_link({local, ?SERVER}, ?MODULE, []).
+
+init([]) ->
+    SupFlags = #{
+        strategy => one_for_all,
+        intensity => 0,
```

Comment:
Q: This supervisor doesn't allow any failures, what is the escalation strategy?

Reply:
Great catch.  This supervisor was auto-generated by rebar, and I didn't noticed this.  It actually doesn't keep any children, so I'll just remove it and the app module to avoid confusion, since they are useless in this case.  Thanks!  :beers: